lib/gtfs_server/agencies/trimet/client.ex

```diff
@@ -0,0 +1,49 @@
+defmodule GtfsServer.Agencies.Trimet.Client do
+  use HTTPoison.Base
+
+  @endpoint "http://developer.trimet.org/ws/V1"
+  @trimet_app_id Application.get_env(:gtfs_server, :trimet_app_id)
+
+  alias GtfsServer.Protobufs.FeedMessage
+  alias GtfsServer.Agencies.Trimet.Client
+
+  require Logger
+
+  @moduledoc """
+  GtfsProducer.TrimetRT service is responsible for wrapping the Trimet GTFS-RT
+  API so that Trimet GTFS-RT data can be injested to the platform.
+  """
+  def get_feed(:trimet_vehicle_posistion_feed) do
+    with {:error, message} <- fetch("/VehiclePositions")  do
+      Logger.error("Trimet GTFS RT feed: #{message}")
+    end
+  end
+
+  def get_feed(:trimet_alert_feed) do
+    with {:error, message} <- fetch("/FeedSpecAlerts")  do
+      Logger.error("Trimet GTFS RT feed: #{message}")
+    end
+  end
+
+  def fetch(endpoint) do
+    case Client.get(endpoint, [], hackney: [pool: :trimet_gtfs_pool]) do
```

Comment:
Be very carful with open ended naming like this, without context I have to keep referring to top of file alias, a better solution is to just alias Trimet and call Trimet.Client which instantly gives you context. When you hit abstracted names that can go either way, leaving the module 1 level up on is generally preferred for context